<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>342</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>342</width>
    <height>281</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Harf Notunu Hesaplayıcı</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>321</width>
     <height>261</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>321</width>
     <height>261</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="sinif_ortalamasi_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>117</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sınıf Ortalaması :</string>
    </property>
   </widget>
   <widget class="QLabel" name="hamnot_label_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>200</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ham Not :</string>
    </property>
   </widget>
   <widget class="QLabel" name="hamnot_label">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>200</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hesapla_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="sonuc_label_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>230</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sonuç :</string>
    </property>
   </widget>
   <widget class="QLabel" name="sonuc_label">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>230</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="ybn_label_2">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>200</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>YBN :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ybn_label">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>200</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="temizle_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Temizle</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="sinav_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Sınav</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="odev_label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>50</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ödev :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Quiz :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="final_label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>110</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Final :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="vize_label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>80</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Vize :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="yuzde_groupBox">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>10</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Yüzde</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="quiz_yuzde_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="odev_yuzde_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="vize_yuzde_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="final_yuzde_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>111</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="not_groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Not</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="quiz_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="odev_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="final_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>111</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="vize_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="ortalama_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>161</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
